### PR TITLE
Fix git pull --ff-only syntax.

### PR DESCRIPTION
**Pull request by jankaluza**
Signed-off-by: Jan Kaluza <jkaluza@redhat.com>

### CODE REVIEW

**Review by sturivny — APPROVED** [review]
LGTM

File: lib/distrobaker.py [hunk 1]
```diff
@@ -470,7 +470,7 @@ def sync_repo(comp, ns='rpms', nvr=None):
     else:
         logger.debug('Attempting to synchronize the %s/%s branches using the clean pull mechanism.', ns, comp)
         try:
-            repo.git.pull('--ff-only', bscm['ref'])
+            repo.git.pull('--ff-only', 'source', bscm['ref'])
```
Comment (sgallagher) [inline lib/distrobaker.py:473]: Can we have this as a variable so if we decide we need to change the remote name for some reason, it only needs to happen in one place?

Comment (jankaluza) [reply lib/distrobaker.py:473]: We can, but does it need to happen as part of this PR? :)

Comment (sgallagher) [reply lib/distrobaker.py:473]: It does not.

**Pull request merged**